### PR TITLE
adding retries for AE and sentry exception logging

### PR DESCRIPTION
**Pull request by ch40s3k**

### CODE REVIEW

File: alamo_alerter/logger.py [hunk 1]
```diff
@@ -20,11 +20,16 @@
                 'level': 'DEBUG',
                 'class': 'logging.StreamHandler',
                 'formatter': 'verbose'
+            },
+            'sentry': {
+                'level': 'CRITICAL',
+                'class': 'raven.handlers.logging.SentryHandler',
+                'dsn': 'https://b6b175ec6e8c41c3bcaaa4a118287c2d:343346720d3f4be9a53a6f38fc3222c2@sentry.allegrogroup.com/71?verify_ssl=0'  # noqa
```
Comment (deejay1) [inline alamo_alerter/logger.py]: OMGWTFBBQ

Comment (ch40s3k) [inline alamo_alerter/logger.py]: Find solution :)

**Comment by deejay1** [comment]
:+1:

**Comment by h00dy** [comment]
:+1:

**Pull request merged**